src/styles/index.css

```diff
@@ -0,0 +1,393 @@
+body {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  width: calc(100px * 7 + 1px * 6 + (10px));
```

Comment:
The width calculation seems to be incorrect. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the arithmetic operations inside `calc` should be surrounded by spaces. Moreover, the `+ (10px)` part seems redundant and does not follow a clear logic for the calendar width.